catalog/ingress-nginx/releases.yaml

```diff
@@ -0,0 +1,36 @@
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: ingress-nginx
+  namespace: ingress-nginx
+spec:
+  interval: 10m0s
+  chart:
+    spec:
+      chart: ingress-nginx
+      version: v4.4.2
+      sourceRef:
+        kind: HelmRepository
+        name: ingress-nginx
+      interval: 1m
+  install:
+    crds: CreateReplace
+  upgrade:
+    crds: CreateReplace
+  values:
+    controller:
+      metrics:
+        enabled: true
```

Comment:
@kharf should we remove these `values` and let the user "override" them as they needed?

Reply:
@beiertu-mms yea probably, if many users need them, they can set up and make a template for that